src/Dropdown/Dropdown.tsx

```diff
@@ -14,12 +14,14 @@ export interface DropdownProps {
   onChange: ChangeEventHandler;
   /** The name of the dropdown */
   name: string;
-  /** An array of objects with the properties 'label' and 'value' for each dropdown option */
-  options: Array<{label: string; value: string}>;
+  /** An array of objects with the properties 'label', 'value', and 'disabled' for each dropdown option */
+  options: Array<{label: string; value: string; disabled?: boolean}>;
```

Comment:
I would extract this type out on it's own, or better yet - type this as [`HTMLOption[]`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOptionElement)

Reply:
Ok good idea - I extracted this type out through 2f0f88b.

Reply:
When I tried to type it as `HTMLOption`, it made all of the properties under HTMLOption mandatory, so I ended up extracting the type out instead

Reply:
Ah, that's frustrating. Fair enough :)